test/elixir_analyzer/exercise_test/assert_call/erlang_modules_test.exs

```diff
@@ -0,0 +1,154 @@
+defmodule ElixirAnalyzer.ExerciseTest.AssertCall.ErlangTest do
+  use ElixirAnalyzer.ExerciseTestCase,
+    exercise_test_module: ElixirAnalyzer.Support.AnalyzerVerification.AssertCall.Erlang
+
+  test_exercise_analysis "Calling :rand.normal in function/0",
+    comments: [
+      "found a call to :rand in function/0",
+      "found a call to :rand in module",
+      "found a call to :rand.normal in function/0",
+      "found a call to :rand.normal in module"
+    ] do
+    [
+      defmodule AssertCallVerification do
+        def function() do
+          r = :rand.normal()
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias :rand, as: Rand
+
+        def function() do
+          r = Rand.normal()
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias :rand
+
+        def function() do
+          r = :rand.normal()
+        end
+      end,
+      defmodule AssertCallVerification do
+        def function() do
+          r = :rand.normal()
+        end
+      end
```

Comment:
Isn't this snippet exactly the same as the first one in this list?

Reply:
Indeed, I'll get rid of it. Maybe it was something else before I made changes to `import`.